core/storage/statistics/gae_models.py

```diff
@@ -740,6 +964,20 @@ class StateAnswersCalcOutputModel(base_models.BaseMapReduceBatchResultsModel):
     @classmethod
     def create_or_update(cls, exploration_id, exploration_version, state_name,
                          calculation_id, calculation_output):
+        """Creates or updates StateAnswersCalcOutputModel and then writes
+        it to the datastore.
+
+        Args:
+            exploration_id: str. ID of the exploration currently being played.
+            exploration_version: int. Version of exploration.
+            state_name: str. Name of current state.
+            calculation_id: str. ID of the calculation performed.
+            calculation_output: dict. Output of the calculation which is to be
+            stored as a Json blob.
```

Comment:
Indent by four

Reply:
Thanks for reviewing. I made changes to this one.